### ISSUE TITLE
Eternal ling horror form, keeping ability to wear, wield and use anything, but turn mute. Also how to become truly invisible, even to rightclicks

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#34171]
(WEB REPORT BY: securitymain REMOTE: 206.221.180.138:7777)
# Revision
c19080349e9fa02cd6cce1a6412013dd1e590c06

# Description
This is how to become an unkillable terror machine as a ling

# Steps to Reproduce
This is all the circumstances, but the actual cause is probably just a part of it

1. Be on lamprey
2. Clown
3. Traitor
4. Lingform gained from vial
5. Invisible sprayed body and backpack
6. Stacked into the stacky-wheelchair
7. Turn into horror form (NOW MUTE)
8. Wait until timer runs out, notice random body, no longer invisible
9. Transform into yourself
10. Eternal ling-human hybrid form. Human part of sprite is invisible as before. Can wear hardsuits, use guns, whatever.

BONUS! HOW TO BECOME INVISIBLE TO RIGHTCLICKS!

11. Be on stacky chair as horror-human hybrid
12. Have someone RPG the chair
13. Your sprite is now stuck as high as it was on the chair, even if off-screen. Invisible! Unclickable!

# What you Expected
Whacky behavior when using the stacky chair and the glitchy horror form

# What Actually Happened
Exactly that

**Comment by pedr69** [comment]
yeah the chair and weldedlocker-resisting pixel offsets can result in wacky stuff
the randombody part seems interesting though, horrorform shouldn't do that

**Comment by Exxion** [comment]
I think your sprite has to be at least like nine tiles offscreen to render you truly invisible from all perspectives, but you are basically unclickable as long as it's more than one tile offscreen
If we didn't use `TILE_BOUND`, your sprite would be visible to people around it no matter how far it was offset from your body, but we do use `TILE_BOUND`, so people can only see it if their view of your actual position is unobstructed and no more than two tiles offscreen or something

I have no idea about the randombody, mute, semi-horrorform part
